crates/sui/src/validator_commands.rs

```diff
@@ -554,7 +554,9 @@ async fn call_0x5(
         .quorum_driver()
         .execute_transaction_block(
             transaction,
-            SuiTransactionBlockResponseOptions::full_content(),
+            SuiTransactionBlockResponseOptions::new()
+                .with_input()
+                .with_effects(),
```

Comment:
are balance_changes & object_changes offending?

Reply:
yes, fetching `balance_change` will trigger an error on fullnode as mentioned in the summary